theano/scan_module/scan_opt.py

```diff
@@ -379,7 +407,10 @@ def process_node(self, fgraph, node):
 # This is a global opt for historical reason
 # It should be possible to change it to a local opt.
 class PushOutSeqScan(gof.Optimizer):
-
+    """
+    A global optimizer for pushing out the input variables that are not being
+    used inside the scan and provided in the sequences.
```

Comment:
@nouiz I have fixed the both of the docstrings that you have pointed out, there:

https://github.com/Theano/Theano/pull/3216